packages/yourloops/test/unit/app/main-lobby.test.tsx

```diff
@@ -25,148 +25,94 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-import React, { FunctionComponent } from 'react'
-import { createMemoryHistory, MemoryHistory } from 'history'
-import { Router } from 'react-router-dom'
-import * as auth0Mock from '@auth0/auth0-react'
-import { Auth0Provider } from '@auth0/auth0-react'
-
-import * as authHookMock from '../../../lib/auth'
-import { MainLobby } from '../../../app/main-lobby'
-import renderer, { ReactTestRenderer } from 'react-test-renderer'
-import { ConsentPage } from '../../../pages/login'
-import CompleteSignUpPage from '../../../pages/signup/complete-signup-page'
 import User from '../../../lib/auth/user'
-import PatientConsentPage from '../../../pages/patient/patient-consent'
-import DirectShareApi from '../../../lib/share/direct-share-api'
-import TrainingPage from '../../../pages/training/training'
-import LoginPage from '../../../pages/login/login-page'
+import { getRedirectUrl } from '../../../app/main-lobby'
 
-jest.mock('../../../lib/auth')
-jest.mock('@auth0/auth0-react')
 describe('Main lobby', () => {
```

Comment:
👏🏻 speed up !

Reply:
This was our longest unit test. Now it runs in 3ms